operate/wallet/master.py

```diff
@@ -392,26 +476,6 @@ def transfer(
         rpc: t.Optional[str] = None,
     ) -> t.Optional[str]:
         """Transfer funds to the given account."""
```

Comment:
@OjusWiZard  Deprecate transfer_asset and enhance transfer with default asset=ZERO_ADDRESS parameter ?